docs/environment.yml

```diff
@@ -0,0 +1,11 @@
+# File: docs/environment.yaml
+
+name: docs
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - sphinx==4.2.0
+  - pip:
+    - sphinx_rtd_theme>=0.2.4
+    - recommonmark
```

Comment:
Just out of curiosity, why `pip` for the other two dependencies? I think they're available on `conda-forge`, so you might be able to get away with a pure conda docs build, which minimizes [mixing conda and pip installs](https://www.anaconda.com/blog/using-pip-in-a-conda-environment).

* https://anaconda.org/conda-forge/sphinx_rtd_theme
* https://anaconda.org/conda-forge/recommonmark

Reply:
I wasn't aware they're available on conda-forge; I had set up this environment file last week (I think by looking at another template) and just didn't change it. I can do so if you want to request it.

Reply:
```suggestion
# File: docs/environment.yaml

name: docs
channels:
  - conda-forge
  - defaults
dependencies:
  - sphinx==4.2.0
  - sphinx_rtd_theme>=0.2.4
  - recommonmark
```